en/getting_started/create.md

```diff
@@ -2,15 +2,15 @@
 
 Consul git repo is hosted at Github.com, we recommend using it for your fork's repo to make things easier. But you can use any other service like Bitbucket or Gitlab if you want to, just don't forget to put a reference link back to CONSUL on the footer to comply with project's license (GPL Affero 3).
 
-1. [Register an user account on Github](https://github.com/join) if you don't have one
+1. [Register an user account on Github](https://github.com/join) if you don't have one.
 
-2. [Create an Organization](https://help.github.com/articles/creating-a-new-organization-from-scratch/) on Github with the name of your city or the organization that's going to use Consul. **This is not mandatory**, but it will help undertand the fork's purpose and future contributions by other users.
+2. [Create an Organization](https://help.github.com/articles/creating-a-new-organization-from-scratch/) on Github with the name of your city or the organization that's going to use Consul. **This is not mandatory**, but it will help underhand the fork's purpose and future contributions by other users.
```

Comment:
Perhaps the original text meant to try to say "understand". 
I don't know if `underhand` makes more sense than `understand` in this case 🤔

Reply:
No, it was my mistake. I'll fix it.